SimTKcommon/Simulation/include/SimTKcommon/internal/MeasureImplementation.h

```diff
@@ -1289,6 +1289,43 @@ template <class T>
 class Measure_Differentiate_Result {
 public:
     Measure_Differentiate_Result() : derivIsGood(false) {}
+
+    Xml::Element toXmlElement(const std::string& name) const {
+        static const int version = 1;
+        Xml::Element e("Measure_Differentiate_Result");
+        if (!name.empty()) e.setAttributeValue("name", name);
+        e.setAttributeValue("version", String(version));
+        e.appendNode(toXmlElementHelper(operand, "operand", true));
+        e.appendNode(toXmlElementHelper(operandDot, "operandDot", true));
+        e.appendNode(toXmlElementHelper(derivIsGood, "derivIsGood", true));
+        return e;
+    }
+
+    void fromXmlElement(Xml::Element e, 
+                        const std::string& requiredName="") {
+        const int versionInXml = e.getRequiredAttributeValueAs<int>("version");
+        SimTK_ERRCHK1_ALWAYS(versionInXml == 1,
+            "Measure_Differentiate_Result::fromXmlElement",
+            "Expected Measure_Differentiate_Result version=1 but got %d.",
+            versionInXml);
+        if (!requiredName.empty()) {
+            const String& name = e.getElementTag();
```

Comment:
Shouldn't this be `e.getOptionalAttributeValue("name");`? This occurs elsewhere too.